BoxContentSDK/BoxContentSDK/External/HashHelper/BOXStreamingHashHelper.h

```diff
@@ -0,0 +1,18 @@
+//
+//  BOXStreamingHashHelper.h
+//  BoxContentSDK
+//
+//  Created by Prithvi Jutur on 5/21/18.
+//  Copyright © 2018 Box. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol BOXStreamingDataProcessor <NSObject>
```

Comment:
Maybe a short doc string. Otherwise I have to look elsewhere to see why `close` returns `NSString *`